README.md

```diff
@@ -226,6 +226,15 @@ index them run:
 make reindex
 ```
 
+### 2.6 Cancelling all ongoing (re)indexing operations
+
+
+1) Go to the Simple Queue Service dashboard in the AWS Console. Then, find your target notify SQS queue (should be named azul-notify-...) and purge the queue.
+2) Go to the Lambda dashboard in the AWS Console. Find the `azul-indexer...-write` lambda. Then, disable the event binding to the document queue (usually named `azul-documents-...`). This is done by clicking on the `SQS` trigger in the *Designer* box, clicking on the *Enabled* switch of `azul-documents-...` in the newly appeared *SQS* box, then finally saving your settings.
+3) Purge the remaining queues.
+4) Renable the event binding from step 2.
```

Comment:
Swap 4 and 5.

Reply:
By swapping the steps 4 and 5, repeating step 2 without reenabling the event won't have any effect. So in step 4, I changed instructions so it says to repeat step 3 after waiting for 5 minutes for the queues to empty.

Reply:
Other than the above, steps 4 and 5 are still swapped.